### DISCUSSION TITLE
Query requests or suggestions

### DISCUSSION THREAD

**Discussion by OrdepA17** [uprm-inso4101-2023-2024-S1/semester-project-tutoring-app#108] [Ideas]
Hey @everyone, I have opened this discussion so that all the teams can request specific queries that they believe their tasks require. Team 4 will not be the only team working on the API connections but we are willing to offer our knowledge and assistance with the matter.  Feel free to reach out if there any concerns.

**Comment by Jonathan-Rod** [comment]
The search bar component currently uses a constant array for its contentList property. However, we require access to API endpoints that provide 'getters' to obtain an array of data that updates periodically. This will allow us to have up-to-date information.
